### PR TITLE
NMA-1220

### PR DESCRIPTION
**Pull request by hadia**
As a user who wants to cancel their transaction, I would like to make sure I retain the associated Dash in my Dash Wallet.

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

**Review by Syn-McJ — APPROVED** [review]
Looks good

File: integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseConvertCryptoFragment.kt [hunk 1]
```diff
@@ -273,20 +260,78 @@ class CoinbaseConvertCryptoFragment : Fragment(R.layout.fragment_coinbase_conver
...
+        }
+    }
+
+    @SuppressLint("SetTextI18n")
+    private fun setMinAmountErrorMessage() {
+        convertViewModel.selectedLocalExchangeRate.value?.let { rate ->
+            selectedCoinBaseAccount?.currencyToDashExchangeRate?.let { currencyToDashExchangeRate ->
+                val cleanedValue =
+                    (
+                        "0.01909299".toBigDecimal() /
```
[the remark this answers is not in the document]
Comment (HashEngineering) [reply integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseConvertCryptoFragment.kt:324]: Yes, a constant with a descriptive name would help a lot.  Is this a minimum value in DASH that can be entered?

**Review by HashEngineering — APPROVED** [review]

File: integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseConvertCryptoFragment.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (hadia) [reply integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseConvertCryptoFragment.kt]: Done

File: integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt [hunk 2]
```diff
@@ -192,6 +195,10 @@ class ConvertViewViewModel @Inject constructor(
         _dashToCrypto.value = dashToCrypto
     }
 
+    fun clear() {
```
Comment (ClaudeHangui) [inline integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt:198]: You don't actually need to handle this clearing in the view's `onDestroy()` method: you can do this directly within the view model's `onCleared()`methods

**Review by ClaudeHangui — APPROVED** [review]
A small observation regarding the usage of `onCleared()` of the view model; but all things considered, this is ok

**Pull request merged**